### DISCUSSION TITLE
vscode intellisense for shortcuts

### DISCUSSION THREAD

**Discussion by andrewdoro** [unocss/unocss#2053] [General]
Is there any way to make intellisense work for shortcuts in the config file?
![image](https://user-images.githubusercontent.com/17547035/211171390-de2f2de9-6644-41a8-9acf-ddaabbd8f777.png)

**Comment by zyyv** [comment]
<img width="520" alt="image" src="https://user-images.githubusercontent.com/42139754/211231041-195bb48a-a321-41e0-8353-ff9eda2a690c.png">

It works well in my project.

**Reply by andrewdoro** [reply]
I was thinking if it was possbile to get something like this when I create them inside the shortcuts object in the config file
![image](https://user-images.githubusercontent.com/17547035/211260997-9a92dcb7-d179-4ab2-a56c-34f73c7b2042.png)
